etcd-env.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.bindir      = 'exe'
   spec.executables = spec.files.grep(/^exe\//) { |f| File.basename(f) }
 
-  spec.add_development_dependency 'bundler', '~> 1.9'
+  spec.add_development_dependency 'bundler', '~> 1.7'
```

Comment:
We shouldn't change our app dependencies to make circleci happy. Can't we just install the latest bundler in circleci?